superset-frontend/src/explore/controlUtils/getControlValuesCompatibleWithDatasource.ts

```diff
@@ -23,8 +23,10 @@ import {
   isAdhocMetricSimple,
   isSavedMetric,
   isSimpleAdhocFilter,
+  isFreeFormAdhocFilter,
   JsonValue,
   SimpleAdhocFilter,
+  FreeFormAdhocFilter,
```

Comment:
do you still need this import for `FreeFormAdhocFilter`?

Reply:
nope removed it